patches/@lion+form-core+0.16.0.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/@lion/form-core/src/validate/ValidateMixin.js b/node_modules/@lion/form-core/src/validate/ValidateMixin.js
+index ea18839..c4bfde2 100644
+--- a/node_modules/@lion/form-core/src/validate/ValidateMixin.js
++++ b/node_modules/@lion/form-core/src/validate/ValidateMixin.js
+@@ -103,8 +103,8 @@ export const ValidateMixinImplementation = superclass =>
+       return {
+         ...super.slots,
+         feedback: () => {
+-          // @ts-ignore we load a polyfill to support createElement on shadowRoot
+-          const feedbackEl = this.shadowRoot.createElement('lion-validation-feedback');
++          const root = this.shadowRoot.createElement ? this.shadowRoot : document;
++          const feedbackEl = root.createElement('lion-validation-feedback');
```

Comment:
this is here to make the portal search not break => once this is released and lion updated we can remove this and use the latest lion versions